### PR TITLE
Javascript/javascript1/week3

### PR DESCRIPTION
**Pull request by islam-fawzy25**

### CODE REVIEW

**Review by aina21 — CHANGES_REQUESTED** [review]
Nice job :+1:

File: javascript/javascript1/week3/homework.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+
+  function timeOfTravel(){
+    let travelResult = travelInformation.destinationDistance / travelInformation.speed ;
+    let travelHistory = travelResult +'';
+    let travel = new Array();
```
[the remark this answers is not in the document]
Comment (islam-fawzy25) [reply javascript/javascript1/week3/homework.js:36]: i did like i found at somewhere " let travel = new Array();
     travel = travelHistory.split('.'); "
but its good short cut 
thanks

File: javascript/javascript1/week3/homework.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    travel = travelHistory.split('.');
+    let  minutesCalculation = travel[1] *60/100;
+     travelTime= travel[0]+ ' hours '+ minutesCalculation +' minutes'
```
[the remark this answers is not in the document]
Comment (islam-fawzy25) [reply javascript/javascript1/week3/homework.js:39]: sorry i can not got it ,

File: javascript/javascript1/week3/Step3.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+saveNote("Do laundry", 2);
+
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+// Get a note
+
+function getNote(id){
+  // your code here
+ for(let i=0; i< notes.length; i++){ 
+   if ( notes[i].id == id ){
```
[the remark this answers is not in the document]
Comment (islam-fawzy25) [reply javascript/javascript1/week3/Step3.js:22]: you mean true as boolean or true as variable ?
i tried to do that as boolean and it give Error
Screenshot (9)

File: javascript/javascript1/week3/homework.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  // Write some code here
+for (let i = 0; i < names.length; i++) {
+    if(nameToRemove === names[i]){ 
+        names.splice(names.indexOf (nameToRemove),1)
```
[the remark this answers is not in the document]
Comment (islam-fawzy25) [reply javascript/javascript1/week3/homework.js:19]: i can not got it 
i removed for loop , so i did not remove the nameToRemove 
Screenshot (10)

[the remark this answers is not in the document]
Comment (islam-fawzy25) [reply javascript/javascript1/week3/homework.js:19]: perfect , thank you

File: javascript/javascript1/week3/Step3.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (islam-fawzy25) [reply javascript/javascript1/week3/Step3.js:22]: got it

**Pull request merged**